### PR TITLE
Update to API 5.18

### PR DESCRIPTION
**Pull request by skoef**
### Short description
implemented `ConvertVpsBackupToSnapshot` to update to API 5.18

### Checklist

- [x] I read the [CONTRIBUTING.md](https://github.com/transip/gotransip/blob/master/CONTRIBUTING.md) document
- [x] This code actually compiles
- [x] This code solves my problem
- [x] I've updated the inline documentation
- [x] I added or modified test(s) to prevent this issue from ever occuring again

### CODE REVIEW

File: vps/api.go [hunk 1]
```diff
@@ -596,3 +596,16 @@ func GetAvailableAvailabilityZones(c gotransip.Client) ([]AvailabilityZone, erro
...
+
+// ConvertVpsBackupToSnapshot converts a backup to a separate snapshot for a VPS
+func ConvertVpsBackupToSnapshot(c gotransip.Client, vpsName, description string, vpsBackupID int64) error {
+	sr := gotransip.SoapRequest{
+		Service: serviceName,
+		Method:  "convertVpsBackupToSnapshot",
+	}
+	sr.AddArgument("vpsName", vpsName)
+	sr.AddArgument("description", description)
+	sr.AddArgument("vpsBackupID", fmt.Sprintf("%d", vpsBackupID))
```
Comment (svenwiltink) [inline vps/api.go]: not sure if it should be ID or Id. Or if it matters at all

Comment (skoef) [reply vps/api.go]: We're already using `vpsBackupID` in `RevertVpsBackup` (https://github.com/transip/gotransip/blob/master/vps/api.go#L397) so this keeps it consistent. Also, golint will complain about it: `func parameter vpsBackupId should be vpsBackupID (golint)`

Comment (svenwiltink) [reply vps/api.go]: The string argument to `AddArgument` is with Id though:
```
sr.AddArgument("vpsBackupId", fmt.Sprintf("%d", vpsBackupID))
```

Comment (skoef) [reply vps/api.go]: true but one is the go variable and the other is the variable it represents in the SOAP request

Comment (skoef) [reply vps/api.go]: You're completely right, that's bad copy/pasting probably.

**Pull request merged**